Extensions/TileMap/helper/TileMapHelper.js

```diff
@@ -1,2 +1,2 @@
lear();try{for(var f=n(e.getLayers()),h=f.next();!h.done;h=f.next()){var y=h.value;if("index"===r&&l!==y.id||"visible"===r&&!y.isVisible())return;if(y instanceof o){var g=y;try{for(var v=(d=void 0,n(g.objects)),x=v.next();!x.done;x=v.next()){var F=x.value,w=i.findTileTexture(F.getTileId(),F.isFlippedHorizontally(),F.isFlippedVertically(),F.isFlippedDiagonally());w&&c.addFrame(w,F.x,F.y-g.tileMap.getTileHeight())}}catch(t){d={error:t}}finally{try{x&&!x.done&&(u=v.return)&&u.call(v)}finally{if(d)throw d.error}}}else if(y instanceof p)for(var T=y,b=T.tileMap.getTileWidth(),_=T.tileMap.getTileHeight(),m=T.tileMap.getDimensionX(),M=T.tileMap.getDimensionY(),H=0;H<M;H++)for(var D=0;D<m;D++){var I=b*D,V=_*H,z=T.get(D,H);if(void 0!==z){var L=i.findTileTexture(z,T.isFlippedHorizontally(D,H),T.isFlippedVertically(D,H),T.isFlippedDiagonally(D,H));if(L){var j=c.addFrame(L,I,V),k=T.tileMap.getTileDefinition(z);k&&k.getAnimationLength()>0&&j.tileAnimX(b,k.getAnimationLength())}else console.warn("Unknown tile id: ".concat(z," at (").concat(D,", ").concat(H,")"))}}}}catch(t){a={error:t}}finally{try{h&&!h.done&&(s=f.return)&&s.call(f)}finally{if(a)throw a.error}}}},t.updatePixiCollisionMask=function(t,e,i,r,l,o,a,s){var d,u,c,f;if(t){t.clear(),t.lineStyle(r,l,o),t.drawRect(0,0,e.getWidth(),e.getHeight());try{for(var h=n(e.getLayers()),y=h.next();!y.done;y=h.next()){var g=y.value,v=e.getTileWidth(),x=e.getTileHeight();if(g instanceof p)for(var F=g,w=0;w<F.tileMap.getDimensionY();w++)for(var T=0;T<F.tileMap.getDimensionX();T++){var b=v*T,_=x*w,m=F.get(T,w),M=F.isFlippedHorizontally(T,w),H=F.isFlippedVertically(T,w),D=F.isFlippedDiagonally(T,w),I=F.tileMap.getTileDefinition(m);if(I){var V=I.getHitBoxes(i);if(V)try{for(var z=(c=void 0,n(V)),L=z.next();!L.done;L=z.next()){var j=L.value;if(0!==j.length){t.beginFill(a,s);for(var k=0;k<j.length;k++){var C=j[k][0],O=j[k][1];if(D){var X=C;C=O,O=X}M&&(C=v-C),H&&(O=x-O),0===k?t.moveTo(b+C,_+O):t.lineTo(b+C,_+O)}t.closePath(),t.endFill()}}}catch(t){c={error:t}}finally{try{L&&!L.done&&(f=z.return)&&f.call(z)}finally{if(c)throw c.error}}}}}}catch(t){d={error:t}}finally{try{y&&!y.done&&(u=h.return)&&u.call(h)}finally{if(d)throw d.error}}}},t}(),w=function(){function t(){this._tileMapCache=new u,this._textureCacheCaches=new u}return t.getManager=function(e){return e.tileMapCollisionMaskManager||(e.tileMapCollisionMaskManager=new t),e.tileMapCollisionMaskManager},t.prototype.getOrLoadTileMap=function(t,e,i,n,r){var l=e+"|"+i;this._tileMapCache.getOrLoad(l,(function(r){t(e,i,(function(t){if(t){var e=y.load(n,t);r(e)}else r(null)}))}),r)},t.prototype.getOrLoadTextureCache=function(t,e,i,n,r,l){var o=n+"|"+r+"|"+i;this._textureCacheCaches.getOrLoad(o,(function(l){t(n,r,(function(t){if(t){var n=i?e(i):null,r=F.parseAtlas(t,n,e);l(r)}else l(null)}))}),l)},t}();t.EditableTileMap=r,t.EditableTileMapLayer=p,t.PixiTileMapHelper=F,t.TileDefinition=d,t.TileMapManager=w,t.TileTextureCache=v,Object.defineProperty(t,"__esModule",{value:!0})}));
```

Comment:
What change happened here?

Reply:
It's the library rebuild. The only real change is the `continue`.